src/plugins/intel_gpu/src/graph/graph_optimizer/mark_runtime_skippable_nodes.cpp

```diff
@@ -137,6 +137,30 @@ void mark_runtime_skippable_nodes::run(program& p) {
                     }
                 }
 
+                node.can_be_optimized(true);
+                // Set runtime skippable only when the node is set as can_be_optimized finally.
+                node.set_runtime_skippable(true);
+                GPU_DEBUG_TRACE_DETAIL << "[mark_runtime_skippable_nodes] : " << node.id() << " can_be_optimized" << std::endl;
+            }
+        });
+        program_helpers::do_for_types<reorder>(*node, [](reorder_node& node){
+            auto impl_params = node.get_kernel_impl_params();
+            if (node.is_output()
+                || node.has_fused_primitives()
+                || (impl_params->get_input_layout(0).format != impl_params->get_output_layout().format)
+                || (impl_params->get_input_layout(0).data_type != impl_params->get_output_layout().data_type))
+                return;
+
+            // TODO: For now, all reorders with dynamic shape are applied.
+            //       A more detailed pattern will need to be applied later
+            if (node.is_dynamic()) {
+                if (node.get_users().size() != 1)
+                    return;
+
+                // If the user is concatenation with 1 user, priority should be given to in place concat optimization at runtime
+                if (node.have_user_with_type<concatenation>())
+                    return;
```

Comment:
now it is marked as can_be_optimized for a broader possibility, need to unset can_be_optimized as false in do_runtime_skip_reorder when it is not actually unskipapble.

Reply:
Fixed it.

Reply:
I dont see the fix in the do_runtime_skip_Reorder. Where did you fix?